packages/syft/src/syft/service/request/request.py

```diff
@@ -592,7 +641,14 @@ def _run(
                     return enclave_res
                 self.linked_obj.update_with_context(context, res)
             else:
-                raise NotImplementedError
+                res = self.mutate(obj, context, revert=True)
+                if res.is_err():
+                    return res
+
+                res = res.ok()
+
+                # TODO: Handle Enclave approval.
```

Comment:
Skipping handling enclave stuff. I will have a word with Rasswanth tomorrow, if there could be a quick for it.